internal/manager/run.go

```diff
@@ -54,8 +54,24 @@ func Run(ctx context.Context, c *Config, diagnostic util.ConfigDumpDiagnostic, d
 	if err != nil {
 		return fmt.Errorf("get kubeconfig from file %q: %w", c.KubeconfigPath, err)
 	}
+
+	setupLog.Info("starting standalone health check server")
+	healthHandler := &healthCheckHandler{}
+	healthHandler.setHealthzCheck(healthz.Ping)
+	go func() {
+		server := &http.Server{
+			Addr:              c.ProbeAddr,
+			Handler:           healthHandler,
+			ReadHeaderTimeout: 3 * time.Second,
+		}
+		err := server.ListenAndServe()
+		if err != nil {
+			setupLog.Error(err, "healthz server failed")
+		}
+	}()
```

Comment:
WDYT about putting this code into the handler itself? Then we could rename `healthHandler` to something like `healthServer` and add a method to starting the server in its own goroutine.

This will make the code at the call site less verbose and it will encapsulate the server launch code from here. WDYT?

Reply:
moved to `health_check.go`